codegens/nodejs-axios/lib/parseRequest.js

```diff
@@ -0,0 +1,194 @@
+var _ = require('./lodash'),
+  sanitize = require('./util').sanitize;
+  // path = require('path');
+
+
+/**
+ * Parses URLEncoded body from request to axios syntax
+ *
+ * @param {Object} body URLEncoded Body
+ * @param {boolean} trim trim body option
+ * @param {boolean} ES6_enabled ES6 syntax option
+ */
+function parseURLEncodedBody (body, trim, ES6_enabled) {
+  var varDeclare = ES6_enabled ? 'const' : 'var',
+    bodySnippet = varDeclare + ' qs = require(\'qs\');\n',
+    dataArray = [];
+
+  _.forEach(body, function (data) {
+    if (!data.disabled) {
+      dataArray.push(`'${sanitize(data.key, trim)}': '${sanitize(data.value, trim)}'`);
+    }
+  });
+  bodySnippet += varDeclare + ` data = qs.stringify({\n ${dataArray.join(',\n')} \n});`;
+  return bodySnippet;
+}
+
+/**
+ * Parses Formdata from request to axios syntax
+ *
+ * @param {Object} body FormData body
+ * @param {boolean} trim trim body option
+ * @param {boolean} ES6_enabled ES6 syntax option
+ */
+function parseFormData (body, trim, ES6_enabled) {
+  var varDeclare = ES6_enabled ? 'const' : 'var',
+    bodySnippet = varDeclare + ' FormData = require(\'form-data\');\n';
+  bodySnippet += varDeclare + ' data = new FormData();\n';
+  // check if there's file
+  const fileArray = body.filter(function (item) { return !item.disabled && item.type === 'file'; });
+  if (fileArray.length > 0) {
+    bodySnippet += varDeclare + ' fs = require(\'fs\');\n';
+  }
+  _.forEach(body, function (data) {
+    if (!data.disabled) {
+      if (data.type === 'file') {
+        // var pathArray = data.src.split(path.sep),
+        //   fileName = pathArray[pathArray.length - 1],
+        var fileContent = `fs.createReadStream("${data.src}")`;
+        // options = `{ knownLength: fs.statSync("${data.src}").size }`;
+        bodySnippet += `data.append('${sanitize(data.key, trim)}', ${fileContent});\n`;
+      }
+      else {
+        bodySnippet += `data.append('${sanitize(data.key, trim)}', '${sanitize(data.value, trim)}');\n`;
+      }
+    }
+  });
+  return bodySnippet;
+}
+
+/**
+ * Parses Raw data to axios syntax
+ *
+ * @param {Object} body Raw body data
+ * @param {boolean} trim trim body option
+ * @param {String} contentType Content type of the body being sent
+ * @param {boolean} ES6_enabled ES6 syntax option
+ */
+function parseRawBody (body, trim, contentType, ES6_enabled) {
+  var varDeclare = ES6_enabled ? 'const' : 'var',
+    bodySnippet = varDeclare + ' data = ';
+  if (contentType === 'application/json') {
+    try {
+      let jsonBody = JSON.parse(body);
+      bodySnippet += `JSON.stringify(${JSON.stringify(jsonBody)});\n`;
+    }
+    catch (error) {
+      bodySnippet += `"${sanitize(body.toString(), trim)}";\n`;
```

Comment:
In the sanitize function, we escape single quotes only. If the body here contains a double quote, it won't get escaped, resulting in a wrong snippet generated. e.g. `"this is a sample "body preview"`. Look at the sanitize functions of other snippets to see how we handle the sanitization of quotes.